tests/yast2_ui/yast2_hostnames.pm

```diff
@@ -0,0 +1,18 @@
+use base "y2x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    x11_start_program("xdg-su -c '/sbin/yast2 host'");
+    if (check_screen "yast2-auth-dialog") {
+        if ($password) {
```

Comment:
I don't get it, do you mean it should move the auth dislog checking and password typing to a function?